### PR TITLE
Refactor usage of os.IsNotExist function

### PR DESCRIPTION
**Pull request by kush-crest**
Fixes https://github.com/elastic/elastic-package/issues/299

Hey,

We have sanity tested the changes done by building the elastic package. 
Do we need any other specific tests to be ran in order to comply?

### CODE REVIEW

**Comment by mtojek** [comment]
@kush-crest Please sign the CLA first, it's a mandatory check.

**Comment by kush-crest** [comment]
Yes i did use same email.
ok, i will push it again.

**Comment by mtojek** [comment]
@kush-crest something is still wrong. I pulled your branch to check commits:

```
commit eed69f0f06ee997663dffef1caaa4c60119a9de5 (HEAD -> draft-pr-testing, kush-crest/draft-pr-testing)
Author: Kush Rana <kushrana_crest@bitbucket.org>
Date:   Wed Aug 11 19:31:12 2021 +0530

    Refactor usage of os.IsNotExist function

    Author:    Kush Rana <kush.rana@crestdatasys.com>
```

It looks like you're using `kushrana_crest@bitbucket.org` address. I assume that you should create these commits with `kush.rana@crestdatasys.com`?

**Comment by kush-crest** [comment]
@mtojek 

My mistake,
Sorry about that.
I rebase it and now i guess its working fine.
Thanks

File: internal/profile/profile.go [hunk 1]
```diff
@@ -76,7 +76,7 @@ func newProfileFromExistingFiles(elasticPackagePath string, profileName string,
 			// if we're treating missing files as soft errors,
 			// just continue on IsNotExist
```
Comment (mtojek) [inline internal/profile/profile.go:77]: Please adjust this comment as it isn't `IsNotExist` anymore.

**Comment by mtojek** [comment]
@kush-crest Please adjust the PR and address left comment.

File: internal/profile/profile.go (hunk 1 above)
Comment (kush-crest) [reply internal/profile/profile.go:77]: Yes Sure

**Review by mtojek — APPROVED** [review]

**Pull request merged**